plugin/core/src/main/java/fr/avianey/androidsvgdrawable/Colorizer.java

```diff
@@ -0,0 +1,135 @@
+package fr.avianey.androidsvgdrawable;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.NamedNodeMap;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import org.xml.sax.SAXException;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * @author dhleong
+ */
+public class Colorizer {
```

Comment:
Fair enough. Will try to find time to add this (and, I guess, rebase to master to fix the CI) sometime